sentry_sdk/transport.py

```diff
@@ -816,11 +1069,18 @@ def make_transport(options: Dict[str, Any]) -> Optional[Transport]:
     ref_transport = options["transport"]
 
     use_http2_transport = options.get("_experiments", {}).get("transport_http2", False)
-
+    use_async_transport = options.get("_experiments", {}).get("transport_async", False)
     # By default, we use the http transport class
-    transport_cls: Type[Transport] = (
-        Http2Transport if use_http2_transport else HttpTransport
-    )
+    if use_async_transport:
```

Comment:
nit on writing good nested/multiple conditionals, each branch should only be responsible for its own logic, so a bit cleaner the other way around

```python
    transport_cls = Http2Transport if use_http2_transport else HttpTransport
    if use_async_transport:
        try:
            asyncio.get_running_loop()
            transport_cls: Type[Transport] = AsyncHttpTransport
        except RuntimeError:
            # No event loop running, fall back to sync transport
            logger.warning("No event loop running, falling back to sync transport.")

```